cumulusci/tasks/robotframework/libdoc.py

```diff
@@ -51,31 +55,50 @@ def _validate_options(self):
                 )
             raise TaskOptionsError(error_message)
 
+    def is_pageobject_library(self, path):
+        """Return True if the file looks like a page object library"""
+        if path.endswith(".py"):
+            with open(path, "r") as f:
+                data = f.read()
+                if re.search(r"@pageobject\(", data):
+                    return True
+        return False
+
     def _run_task(self):
-        libraries = []
-        processed_files = {}
+        kwfiles = []
+        processed_files = []
         for input_file in self.options["path"]:
+            kwfile = KeywordFile(input_file)
             try:
-                libdoc = DocumentationBuilder(input_file).build(input_file)
-                libraries.append(libdoc)
-                processed_files[input_file] = libdoc
-
-                # robot doesn't save the orginal name but we want to use that
-                # in our generated file
-                libdoc.src = os.path.basename(input_file)
-
-                # if we want to save the official libdoc file, uncomment the following
-                # two lines:
-                # libdoc.save(self.options['output_file'], "HTML")
-                # self.logger.info("created {}".format(output_file))
+                if self.is_pageobject_library(input_file):
+                    PageObjects._reset()
+                    Importer().import_class_or_module_by_path(
+                        os.path.abspath(input_file)
+                    )
+
+                    for pobj_name in sorted(PageObjects.registry.keys()):
```

Comment:
```suggestion
                    for pobj_name, pobj in sorted(PageObjects.registry.items()):
```

and then you don't have to do a separate lookup

Reply:
Also I got pretty confused trying to understand this since pobj_name is a tuple but sounds like it's a string